scripts/set_requester_pays.sh

```diff
@@ -23,22 +33,38 @@ ROLE="organizations/${ORG_ID}/roles/RequesterPaysToggler"
 # enable requesterpays permissions
 echo "Enabling permissions for ${USER_EMAIL} to switch on Requester Pays"
 gcloud beta projects add-iam-policy-binding $PROJECT_ID --member=$MEMBER --role=$ROLE | grep -A 1 -B 1 "${MEMBER}"
-
+wait
 # # if needed for troubleshooting, this command retrieves the existing policy
 # gcloud beta projects get-iam-policy $PROJECT_ID | grep -A 1 -B 1 "${MEMBER}"
 
 echo ""
 echo "Gatorcounting for 10 seconds while iam change goes into effect"
 echo ""
-echo "NOTE: if you get an error message saying:"
-echo "    AccessDeniedException: 403 ${USER_EMAIL} does not have storage.buckets.update access to the Google Cloud Storage bucket."
-echo "THEN gatorcount 10 more seconds and run this again."
+echo "NOTE: if you get an error message saying AccessDeniedExeption: 403"
+echo "THEN don't worry, just wait until it shows 'Enabling requester pays."
+echo ""
+echo "retrying in 10 seconds"
 echo ""
 sleep 10
 
+COUNTER=0
+
 for BUCKET in $BUCKETS; do
-  # set requester pays
-  gsutil requesterpays set on ${BUCKET} || exit 1
+  if [[ $BUCKET == *"gs://"* ]]; then
```

Comment:
can we remove the first * here? if the `gs://` is somehow in the middle of the string, we have bigger problems, but we can be reasonably confident that it should be only at the beginning